spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -223,7 +223,6 @@ public RestControllerAdvice restControllerAdvice() {
 
 
 
-	@Configuration
```

Comment:
Again, I may be incorrect.   Probably am...   But should we remove `@Configuration` for these static classes?

Reply:
Ditto https://github.com/spring-cloud/spring-cloud-dataflow/pull/5668/files/7919ace73a567e942b9e0f86bbfa2fca8ca78e27#r1483619438